lib/benchee/system.ex

```diff
@@ -61,19 +61,38 @@ defmodule Benchee.System do
   """
   def cpu_speed, do: cpu_speed(os())
 
-  defp cpu_speed(:macOS), do: system_cmd("sysctl", ["-n", "machdep.cpu.brand_string"])
   defp cpu_speed(:Windows), do: "N/A"
-  defp cpu_speed(:Linux), do: "N/A"
+  defp cpu_speed(:macOS), do: system_cmd("sysctl", ["-n", "machdep.cpu.brand_string"])
+  defp cpu_speed(:Linux) do
+    ["model name\t:" <> cpu_info] = linux_cmd("/proc/cpuinfo", ~r/model name.*:[\w \(\)\-\@\.]*ghz/i)
+    String.trim(cpu_info)
+  end
 
   @doc """
   Returns an integer with the total number of available memory on the machine
   running the benchmarks.
   """
   def available_memory, do: available_memory(os())
 
-  defp available_memory(:macOS), do: system_cmd("sysctl", ["-n", "hw.memsize"])
   defp available_memory(:Windows), do: "N/A"
-  defp available_memory(:Linux), do: "N/A"
+  defp available_memory(:macOS) do
+    {memory, _} = Integer.parse(system_cmd("sysctl", ["-n", "hw.memsize"]))
+    format_memory(memory, 1_000_000_000)
+  end
+  defp available_memory(:Linux) do
+    ["MemTotal:" <> memory] = linux_cmd("/proc/meminfo", ~r/MemTotal.*kB/)
+    {memory, _} = memory
+                  |> String.trim()
+                  |> String.trim_trailing(" kB")
+                  |> Integer.parse
+    format_memory(memory, 1_000_000)
+  end
+
+  defp format_memory(memory, coefficient), do: "#{memory / coefficient} GB"
+
+  defp linux_cmd(file, regex) do
+    Regex.run(regex, system_cmd("cat", [file]))
+  end
```

Comment:
Not sure if `linux_cmd` is the best name here? It's more that linux information is stored in some... form of meta file and then we run a Regex on it to extract the bits that we need, right?

`linux_system_info` or something might sound better but still intention revealing, `linux_cmd` sounds like we run a generic linux command which is not what we do.

And while I'm not a huge fan of comments, a little _why_ comment that maybe explains that bit about meta file and getting the stuff we want. Not sure if it's really needed though - would like your input on that. I mean, I knew it in the back of my head but reading this that we cat a file and then regex it... I guess we don't need the comment but some naming change still sounds pursuit worthy :)